testbeam_analysis/tools/kalman.py

```diff
@@ -335,15 +392,173 @@ def _mat_inverse(X):
     return inv
 
 
+def get_line_intersections_with_plane(line_origins, line_directions,
```

Comment:
Yes, very nice. But unfortunately that is what I have programmed already in the `testbeam_analysis.tools.geometry_utils` module. Check [here](https://github.com/SiLab-Bonn/testbeam_analysis/blob/development/testbeam_analysis/tools/geometry_utils.py#L31). Or is this faster?